tests/commands/test_groups.py

```diff
@@ -296,3 +296,29 @@ def test_groups_group_command__unlink_HTTPError(mock_client, capfd):
     assert "Unlinking group <-> product" in capture.out
     assert "Error" in capture.out
     assert "Unlinked" not in capture.out
+
+
+def test_groups_group_command__migrate(mock_client, capfd):
+    args = Namespace(group_command="migrate")
+    res = groups(args)
+    capture = capfd.readouterr()
+
+    for group in GROUP_RESPONSES:
+        mock_client.migrate_concession_group.assert_any_call(group.id)
+
+    assert res == RESULT_SUCCESS
+    assert "Migrating group" in capture.out
+    assert "Migrated" in capture.out
+
+
+def test_groups_group_command__migrate_HTTPError(mock_client, capfd):
```

Comment:
These tests are good for the `groups()` function and ensuring the args are handled correctly.

Can we also add a few tests for `main()` a la https://github.com/cal-itp/littlepay/blob/main/tests/test_main.py#L114

Reply:
Good catch, thank you! I will add to `test_main`

Reply:
Added a test for simple usage of `littlepay migrate` in [9646562](https://github.com/cal-itp/littlepay/pull/49/commits/9646562c8c476667ca5a047255acbb806210b539)